services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -276,7 +276,7 @@ private void onTick(Date date) {
       if (chainStorageClient.getStore() != null) {
         final Store.Transaction transaction = chainStorageClient.getStore().startTransaction();
         on_tick(transaction, currentTime);
-        transaction.commit();
+        eventBus.post(transaction.commit());
```

Comment:
I believe not posting this event was actually on purpose. On_tick gets called about 4 times a second and only updates the time variable which is useless to update on disk. So I would revert this before merging in this PR.

Reply:
It also updates the justified checkpoint: https://github.com/PegaSysEng/artemis/blob/11c478bdded5bb0c88268ffcd98acd4f16ac4fad/ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/ForkChoiceUtil.java#L215

Reply:
My bad. That was added in 9.2. I forgot about it.